app/models/roster_entry.rb

```diff
@@ -49,7 +73,6 @@ def self.order_for_view(assignment)
         WHEN roster_entries.user_id IN #{sql_formatted_users} THEN 0 /* Accepted */
         ELSE 1                                                       /* Linked but not accepted */
       END
-      , id
```

Comment:
Why was this removed?

Reply:
Pre-PR `order_for_view` would order by accepted/not accepted/not linked _and_ within these three groupings sort by `id`. Due to the way ActiveRecord handles ordering, our `order_for_view.order_by_sort_mode` query would essentially be ordering by accepted/not accepted/not linked, then `id`, then _only if two entities has the same `id`__ order by sort mode, which would mean something would never get sorted by the sort mode. If you'd like, we can explicitly add `id` to the end of this order chain within the controller's full query?

Reply:
> would essentially be ordering by accepted/not accepted/not linked, then id, then, and only if two entities has the same id, order by sort mode

Ah yeah, you're totally right 😄 

> If you'd like, we can explicitly add id to the end of this order chain within the controller's full query to ensure the ordering preference is accepted/not accepted/not linked, sort mode, then id?

This would be good. SQL doesn't guarantee ordering of sets when there is no order, or when there are ties like there will often be here. Adding a `id` order at the end means we won't get assignment repo lists shuffling on reload.